documentation/introduction/gettingStarted/installation/README.md

```diff
@@ -40,6 +40,31 @@ DebiAI offers several installation methods:
 
 - [Python](https://www.python.org/downloads/)
 
+Create a virtual python environment:
+
+```
+python -m venv name_of_virtual-environment
```

Comment:
Change `name_of_virtual-environment` to an image name that would make more sense for a user to write

Reply:
:up